src/components/Drawer/Drawer.js

```diff
@@ -7,19 +7,29 @@ import styles from './Drawer.scss';
 const { block, elem } = bem({
     name: 'Drawer',
     classnames: styles,
-    propsToMods: ['isOpen'],
+    propsToMods: ['isShowAndClose', 'isShowAndOpen', 'isOpen'],
 });
 
 const Drawer = props => {
-    const { title, initialIsOpen, children, ...rest } = props;
-    const [isOpen, setIsOpen] = useState(initialIsOpen);
+    const { title, isShow, isOpen: isOpenProps, initialIsOpen, onClick, children, ...rest } = props;
+    const [isOpen, setIsOpen] = useState(isOpenProps || initialIsOpen);
+    // eslint-disable-next-line no-nested-ternary
+    const status = isShow && isOpen ? 'isShowAndOpen' : isShow ? 'isShowAndClose' : '';
```

Comment:
I would replace this one-liner with a separate function in order to avoid using of double ternary operator and undeclared string constants
```
function getStatus() {
    const STATUS_LIST = {
        IS_SHOW_AND_OPEN: 'IS_SHOW_AND_OPEN',
        IS_SHOW_AND_CLOSE: 'IS_SHOW_AND_CLOSE',
        EMPTY: '',
    };

    if (isShow) {
        return isOpen ? STATUS_LIST.IS_SHOW_AND_OPEN : STATUS_LIST.IS_SHOW_AND_CLOSE;
    }

    return STATUS_LIST.EMPTY;
}
```

Reply:
Thanks, nice improvements.